hash/gameboy.xml

```diff
@@ -10452,6 +10452,19 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="jeepjambp" cloneof="jeepjamb">
+		<description>Jeep Jamboree (USA, prototype)</description>
+		<year>1992</year>
+		<publisher>Virgin Games</publisher>
+		<info name="alt_title" value="Jeep Jamboree - Off Road Adventure (Box)" />
```

Comment:
Descriptive text should not use title case.  Also, do we want to put the box part in the `alt_title` value at all, or just in a comment?  I realise we aren’t completely consistent on this currently.

Reply:
I left it in the alt_title for now. I'm not sure I have an opinion without percolating for a while.